src/about.njk

```diff
@@ -154,9 +154,9 @@
         <p class="text-muted">Birmingham Leader</p>
         <ul class="list-inline social-buttons">
           <li class="list-inline-item">
-            <a href="https://www.linkedin.com/in/stefan-formentano-8809bb93/" target="_blank" class="shadow-sm">
+            <a href="https://www.linkedin.com/in/stefan-formentano-8809bb93/" target="_blank" class="shadow-sm" aria-label="Navigation link to Stefan Formentano LinkedIn Page">
```

Comment:
Screen readers will announce this as a link, so a better aria-label would be "Stefan Formentano's LinkedIn Page"